qualitube/log.py

```diff
@@ -0,0 +1,7 @@
+import logging
+import sys
+
+logger = logging.getLogger('qualitube')
+logger.setLevel('INFO')
+logger.addHandler(logging.FileHandler('pipeline.log'))
+logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
The logging handlers must be left for the user to handle. Otherwise unwanted messages will continue to appear on the terminal. If you want messages to appear only when the user uses the CLI app, handlers should only be added in the `qualitube.main.main()` function.